js/index.js

```diff
@@ -0,0 +1,12 @@
+function login() {
+    let login_account_password = document.querySelector("#password_input").value
+    let retrievedObject = JSON.parse(localStorage.getItem("data"))
+    let account_password_in_LS = retrievedObject.user_data.user_password
+
+    if (login_account_password == account_password_in_LS) {
```

Comment:
As far as I remember, we should compare not the password that is in the localstorage, but the password that will be obtained when decrypting the incoming hash.